src/generated/xproto.rs

```diff
@@ -1252,31 +1254,6 @@ impl TryFrom<u32> for WindowEnum {
 
 /// Opcode for the KeyPress event
 pub const KEY_PRESS_EVENT: u8 = 2;
-/// a key was pressed/released.
-///
-///
-/// # Fields
-///
-/// * `detail` - The keycode (a number representing a physical key on the keyboard) of the key
-/// which was pressed.
-/// * `event_x` - If `same_screen` is true, this is the X coordinate relative to the `event`
-/// window's origin. Otherwise, `event_x` will be set to zero.
-/// * `event_y` - If `same_screen` is true, this is the Y coordinate relative to the `event`
-/// window's origin. Otherwise, `event_y` will be set to zero.
-/// * `root` - The root window of `child`.
-/// * `root_x` - The X coordinate of the pointer relative to the `root` window at the time of
-/// the event.
-/// * `root_y` - The Y coordinate of the pointer relative to the `root` window at the time of
-/// the event.
-/// * `same_screen` - Whether the `event` window is on the same screen as the `root` window.
-/// * `state` - The logical state of the pointer buttons and modifier keys just prior to the
-/// event.
-/// * `time` - Time when the event was generated (in milliseconds).
-///
-/// # See
-///
-/// * GrabKey: request
-/// * GrabKeyboard: request
```

Comment:
This (bad? useless?) docs block was lost.

Reply:
I missed emitting docs for events (and it seems this is the very only event with docs)

Reply:
I think I saw other doc blocks being removed when I scrolled the diff, but I am too lazy to check right now (my browser has some problems when trying to scroll the page with the diff).

Reply:
I'll review it. I think the python generator reordered the field docs alphabetically, whereas this new one keeps the original order from the XML

Reply:
Thanks

Reply:
I tried bringing the XML order to the current code generator, but... well.
xcbgen saves the docs for these fields in a dict and that looses the order. So that's not gonna work.